dot/state/block_finalisation.go

```diff
@@ -191,7 +191,7 @@ func (bs *BlockState) SetFinalisedHash(hash common.Hash, round, setID uint64) er
 		),
 	)
 	if err != nil {
-		return fmt.Errorf("could not send 'notify.finalized' telemetry message, error: %s", err)
+		logger.Debug("could not send 'notify.finalized' telemetry message, error: %s", err)
```

Comment:
Merge development in your branch and use `logger.Debugf` instead, or that will break the dev branch.

nit also maybe a bit shorter:

```suggestion
		logger.Debugf("could not send 'notify.finalized' telemetry message: %s", err)
```

Reply:
done